hasura/migrations/default/1704993161810_create_table_pending_app_metadata/down.sql

```diff
@@ -0,0 +1,6 @@
+
+DROP TABLE "public"."pending_app_metadata";
```

Comment:
@andy-t-wang In your `up` migration you create the table first and `validate_metadata_uris` at the end. The order of actions in the `down` migration should be exactly vice versa.